mava/systems/ppo/ff_ippo.py

```diff
@@ -145,14 +146,15 @@ def _update_minibatch(train_state: Tuple, batch_info: Tuple) -> Tuple:
                 """Update the network for a single minibatch."""
 
                 # UNPACK TRAIN STATE AND BATCH INFO
-                params, opt_states = train_state
+                params, opt_states, entropy_key = train_state
```

Comment:
We need to split and return a new key at each minibatch please. It looks like we are just splitting once at the epoch level and using the same seed for all minibatches.

Reply:
Same applies to all systems :pray:

Reply:
I missed it thank you 🙏